viewer/packages/rendering/src/CadMaterialManager.ts

```diff
@@ -116,13 +116,16 @@ export class CadMaterialManager {
 
   removeModelMaterials(modelIdentifier: string): void {
     const modelData = this.materialsMap.get(modelIdentifier);
-    for (const mat of Object.values(modelData.materials)) {
-      mat.dispose();
-    }
 
-    this.materialsMap.delete(modelIdentifier);
-    modelData.nodeTransformTextureBuilder.dispose();
-    modelData.nodeAppearanceTextureBuilder.dispose();
+    if (modelData) {
```

Comment:
Should it not be an exception if someone calls `removeModelMaterial()` with an invalid model identifier?

Reply:
I guess, yeah

Reply:
Throw exception?

Reply:
Throwing error is bad?

Reply:
? It's not, not in this function at least?

Reply:
And if it is, why check that its "truthy"? Also, modelIdentifier==="" will evaluate to false